charts/fluent-bit/templates/psp.yaml

```diff
@@ -3,6 +3,9 @@ apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
   name: {{ include "fluent-bit.fullname" . }}
+{{- if .Values.podSecurityPolicy.annotations }}
+  annotations: {{- toYaml .Values.podSecurityPolicy.annotations | nindent 4 }}
```

Comment:
```suggestion
  annotations:
    {{- toYaml .Values.podSecurityPolicy.annotations | nindent 4 }}
```

Reply:
I think this suggested change will add an empty line after annotations because of nindent.

Reply:
Should be fine with the `{{-`

Reply:
ok thanks . let me do a fresh PR i forgot to sign one commit and now I messed it up.